docs/src/languages/php/redis.md

```diff
@@ -6,103 +6,49 @@ weight: 7
-        echo "No version of the PhpRedis extension specified.  You must specify a tagged version on the command line."
-        exit 1;
-    fi
-}
-
-ensure_environment
-ensure_arguments "$1"
-run "$1"
-```
-
-2. Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
+A [script](https://github.com/platformsh/snippets/blob/main/src/install-phpredis.sh) is provided to ease the installation of a customer version of PhpRedis.  Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
 hooks:
     build: |
         set -e
-        bash install-redis.sh 5.1.1
+        # Install PhpRedis v5.1.1:
+        curl -fsS https://raw.githubusercontent.com/platformsh/snippets/main/src/install-phpredis.sh | { bash /dev/fd/3 5.1.1 ; } 3<&0
 ```
 
-3. If you ever wish to change the version of PhpRedis you are using, update the build hook and clear the build cache: `platform project:clear-build-cache`.  The new version will *not* be used until you clear the build cache.
+## Install Relay
+
+Relay is a [Redis](../../add-services/redis.md) client like [PhpRedis](https://github.com/phpredis/phpredis) and [PhpRedis](https://github.com/predis/predis). It intents to be a drop-in replacement of those libraries.
+
+That PHP extension is also a shared in-memory cache like APCu. All retrieved keys are held in the PHP master process’ memory, which is shared across all FPM workers.
+
+That means if the FPM Worker #1 fetches the `users:count` key from Redis, the other FPM Workers #2, #3, … will instantaneously retrieve that key from Relay, without having to communicate with Redis.
+
+A [script](https://github.com/platformsh/snippets/blob/main/src/install-relay.sh) is provided to ease the installation of a customer version of PhpRedis.  Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
+## Changing extension of version
+If you ever wish to change the Redis extension or the version you are using, update the build hook and clear the build cache: `platform project:clear-build-cache`.  The new version will *not* be used until you clear the build cache.
 
 There is no need to declare the extension in the `runtime` block.  That is only for pre-built extensions.
 
-## What the script does
+```yaml {location=".platform.app.yaml"}
+hooks:
+    build: |
+        set -e
+        # Install PhpRedis v0.4.3:
+        curl -fsS https://raw.githubusercontent.com/platformsh/snippets/main/src/install-relay.sh | { bash /dev/fd/3 0.4.3 ; } 3<&0
+```
+
+## What those scripts do
 
-1. Downloads the PhpRedis source code.
+1. Downloads the Relay/PhpRedis source code.
 2. Checks out the version specified in the build hook.
 3. Compiles the extension.
 4. Copies the resulting `redis.so` file to [your app root](../../create-apps/app-reference.md#root-directory).
```

Comment:
:clipboard: The scripts are plural

```suggestion
1. Download the Relay/PhpRedis source code.
2. Check out the version specified in the build hook.
3. Compile the extension.
4. Copy the resulting `redis.so` file to [your app root](../../create-apps/app-reference.md#root-directory).
```

Reply:
Not sure if relevant, but Relay's shared object is called `relay.so`.

Reply:
Certainly could make that clearer:

```suggestion
1. Download the Relay/PhpRedis source code.
2. Check out the version specified in the build hook.
3. Compile the extension.
4. Copy the resulting `relay.so`/`redis.so` file to [your app root](../../create-apps/app-reference.md#root-directory).
```